scripts/ismp-get/README.md

```diff
@@ -0,0 +1,37 @@
+# ISMP Get Testing Script
```

Comment:
Would we still need the `ismp-get` scripts in this PR if we separated out the demo stuff into its own branch/draft PR?

Reply:
removed